packages/gamut/src/Form/SelectDropdown.tsx

```diff
@@ -51,13 +56,27 @@ type CustomContainerProps = ContainerProps<OptionStrict, false> & {
   children?: ReactNode[];
 };
 
+const indicatorSizes = {
+  small: {
+    size: 12,
+    icon: MiniChevronDownIcon,
+  },
+  medium: {
+    size: 16,
+    icon: ArrowChevronDownIcon,
+  },
+};
+
 const ChevronDropdown = (props: IndicatorProps<OptionTypeBase, false>) => {
+  const { size } = props.selectProps;
+  const color = props.isDisabled ? 'text-disabled' : 'text';
+  const { icon: IndicatorIcon, ...iconProps } = indicatorSizes[
+    size === 'small' ? 'small' : 'medium'
+  ];
```

Comment:
Ooh even cleaner than what I was thinking of! I like this a lot.

One lil nit:

```suggestion
  const { icon: IndicatorIcon, ...iconProps } = indicatorSizes[size ?? 'small'];
```

Reply:
🙏